src/widget/wmainmenubar.cpp

```diff
@@ -266,8 +266,22 @@ void WMainMenuBar::initialize() {
     QString fullScreenText = tr("Display Mixxx using the full screen");
     auto pViewFullScreen = new QAction(fullScreenTitle, this);
     QList<QKeySequence> shortcuts;
-    shortcuts << QKeySequence::FullScreen;
-    shortcuts << QKeySequence("F11");
+    QKeySequence osShortcut = QKeySequence::FullScreen;
+    QKeySequence mixxxShortcut = QKeySequence("F11");
+
+    // Note(ronso0) In some Linux distros' window managers the OS fullscreen
+    // key sequence is not passed to Mixxx for some reason.
+    // Both adding an empty key sequence or the same sequence twice can render
+    // the fulscreen shortcut unfunctional.
+    // https://bugs.launchpad.net/mixxx/+bug/1882474  PR #3011
+    // 1) Only add the OS shortcut if it's not empty.
```

Comment:
The following code might be more generic and easier to grasp:

```
QList<QKeySequence> shortcuts{QKeySequence("F11")};
// Note(ronso0) In some...
const auto osShortcut = QKeySequence::FullScreen;
if (!osShortcut.isEmpty() && !shortcuts.contains(osShortcut)) {
    shortcuts << osShortcut;
}
```

Reply:
Thanks!